buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchTestBasePlugin.java

```diff
@@ -38,6 +38,7 @@
 import java.util.Map;
 
 import static org.elasticsearch.gradle.util.GradleUtils.maybeConfigure;
+import static org.gradle.util.GFileUtils.mkdirs;
```

Comment:
This is psudo-internal. Can we rely on this?

Reply:
let's do not